web.core.syntax/licenseinfo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/web/core/syntax/completion/resources/class_16.png</file>
+        <file>src/org/netbeans/modules/web/core/syntax/completion/resources/bean_16.png</file>
+        <file>src/org/netbeans/modules/web/core/syntax/completion/resources/function_16.png</file>
+        <file>src/org/netbeans/modules/web/core/syntax/completion/resources/property_16.png</file>
+        <file>src/org/netbeans/modules/web/core/syntax/completion/resources/map_16.png</file>
+        <license ref="Apache-2.0-ASF" />
+        <comment type="COMMENT_UNSUPPORTED" />
+    </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/web/core/syntax/resources/ELExample</file>
+        <file>src/org/netbeans/modules/web/core/syntax/resources/JSPExample</file>
+        <license ref="Apache-2.0-ASF" />
+        <comment type="TEMPLATE_MINIMAL_IP"/>        
```

Comment:
Appropriate comment type is not `TEMPLATE_MINIMAL_IP` but `GUI_USABILITY`. (see https://github.com/apache/incubator-netbeans/pull/679#discussion_r210339082)

Reply:
Done